packages/uikit/src/components/FeeTier/FeeTier.tsx

```diff
@@ -0,0 +1,40 @@
+import BigNumber from "bignumber.js";
+import { forwardRef, useMemo } from "react";
+import styled from "styled-components";
+
+export type FeeTierProps = {
+  type: string;
+  fee: number;
+  denominator?: number;
+  dynamic?: boolean;
+};
+
+export const FeeTier = forwardRef<HTMLSpanElement, FeeTierProps>(
+  ({ type, fee, denominator = 10_000, dynamic }, ref) => {
+    const percent = useMemo(() => {
+      return new BigNumber(fee).div(denominator).times(100).toNumber();
+    }, [fee, denominator]);
+    return (
+      <StyledFeeTier ref={ref}>
+        <span>{type}</span>
+        <span style={{ opacity: 0.5 }}>|</span>
```

Comment:
can we use a Divider UIkit to replace the hard code? It would be better to control the height or other style

Reply:
ok, just Suggestion